tools/comdb2ar/comdb2ar.cpp

```diff
@@ -236,6 +241,14 @@ int main(int argc, char *argv[])
                 mode = CREATE_MODE;
                 break;
 
+            case 'p':
+                mode = PARTIAL_CREATE_MODE;
+                break;
+
+            case 'P':
```

Comment:
p & P could be confusing. I'd go with 'i' for incremental create and 'r' restore.

Reply:
We already have -I.  I was trying to avoid confusion there.

Reply:
FWIW, I don't expect comdb2ar to be used outside of copycomdb2